aws_neuron/assets/dashboards/aws_neuron_overview.json

```
@@ -0,0 +1,1191 @@
+                      "style": {
+                        "palette": "green"
+                      },
+                      "formula": "query1"
+                    },
+                    {
+                      "alias": "Execution error rate",
+                      "style": {
+                        "palette": "red"
+                      },
+                      "formula": "query2"
+                    }
+                  ],
+                  "queries": [
+                    {
+                      "data_source": "metrics",
+                      "name": "query1",
+                      "query": "avg:aws_neuron.execution.status.count{status_type:completed , $instance_id, $instance_type, $availability_zone, $region, $subnet_id} by {instance_id}.as_rate()"
+                    },
+                    {
+                      "data_source": "metrics",
+                      "name": "query2",
+                      "query": "avg:aws_neuron.execution.status.count{not status_type:completed} by {instance_id}.as_rate()"
+                    }
+                  ],
+                  "response_format": "timeseries",
+                  "style": {
+                    "palette": "dog_classic",
+                    "order_by": "values",
+                    "color_order": "shuffled",
+                    "line_type": "solid",
+                    "line_width": "normal"
+                  },
+                  "display_type": "area"
+                }
+              ]
+            },
+            "layout": {
+              "x": 0,
+              "y": 1,
+              "width": 6,
+              "height": 3
+            }
+          },
+          {
+            "id": 245689066689228,
+            "definition": {
+              "title": "Execution latency",
+              "title_size": "16",
+              "title_align": "left",
+              "show_legend": true,
+              "legend_layout": "auto",
+              "legend_columns": [
+                "avg",
+                "min",
+                "max",
+                "value",
+                "sum"
+              ],
+              "type": "timeseries",
+              "requests": [
+                {
+                  "formulas": [
+                    {
+                      "style": {
+                        "palette": "warm"
+                      },
+                      "number_format": {
+                        "unit": {
+                          "type": "canonical_unit",
+                          "unit_name": "second"
+                        }
+                      },
+                      "formula": "query1"
+                    }
+                  ],
+                  "queries": [
+                    {
+                      "data_source": "metrics",
+                      "name": "query1",
+                      "query": "avg:aws_neuron.execution.latency_seconds{$instance_id, $instance_type, $availability_zone, $region, $subnet_id} by {percentile}"
```

```suggestion
                      "query": "avg:aws_neuron.execution.latency_seconds{$instance_id, $instance_type, $availability_zone, $region, $subnet_id} by {percentile, instance_id, instance_type, availability_zone, region, subnet_id}"
```
Maybe something along these lines. As a user looking for information, having all the data aggregated by percentile only seem too broad. But maybe having all these might be too much. But I think you need to include more so that at a glance a user can see there's a problem and then use the template variables to drill down. But having too little context, could lead a user to not realize they need to drill down.